packages/core/stories/card.stories.tsx

```diff
@@ -42,9 +42,9 @@ const Examples = () => (
     <ExampleRow name="Default">
       <Card>
         <div>
-          <h1 style={{ margin: "0", lineHeight: "1.3em" }}>
+          <H1 style={{ margin: "0", lineHeight: "1.3em" }}>
```

Comment:
I haven't checked but are these styles still needed?

Reply:
they were for QA stories, no longer needed (for now anyway)